test-project/build.gradle

```diff
@@ -26,6 +26,43 @@ def setupDependencies(String platform) {
     return "mac" == "$platform" ? "$graalFolder/Contents/Home" : "$graalFolder"
 }
 
+def setupJavaFXDependencies(String platform, String target) {
+    def javafxSdk = new File("${System.properties['user.home']}/.gluon/substrate/javafxStaticSdk")
+    if (!javafxSdk.exists()) {
+        javafxSdk.mkdirs()
+    }
+    def version = "14-ea+gvm1"
+    def name
+    if ("linux" == "$platform") {
+        name = "linux-x86_64"
+    } else if ("mac" == "$platform") {
+        if ("ios" == "$target") {
+            name = "ios-arm64"
```

Comment:
This looks extremely brittle. Can we use the Constants?

Reply:
Indeed, it should be fixed with #56